capa/ida/plugin/form.py

```diff
@@ -566,6 +570,7 @@ def ensure_capa_settings_rule_path(self):
             logger.info("User cancelled analysis.")
             return False
 
+        path = settings.user.get(CAPA_SETTINGS_RULE_PATH, "")
         if not os.path.exists(path):
             logger.error("rule path %s does not exist or cannot be accessed" % path)
             return False
```

Comment:
can we remove this block? the same check happens above

Reply:
good catch; fixed